### PR TITLE
Feature/footer

### LINKED ISSUE #12: Create Footer
As a user, I would like to be able to navigate to Home, the About page, and to this repo.

AC-
- [x] Use footer from [Vuetify](https://vuetifyjs.com/en/components/footer/#company-footer)
- [x] The primary color needs to match that of the header (hex #1976D2)
- [x] Needs to have the year the Life At Home
- [x] Needs to have links to the home page, the About page, and to this repo.
- [x] Needs to Sticky to the bottom

- Design: https://preview.uxpin.com/fa165a3181b8d0c14e4cf54137715fa4f59e76d8#/pages/127018043/simulate/no-panels?mode=i

### PR DESCRIPTION
**Pull request by chrissyalbert**
I removed font awesome and the commented code. I couldn't find the exact external link or open in new icon from the Materials Icon.

Closes #12

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1028,6 +1028,37 @@
...
+  version "5.13.0"
+  resolved "https://registry.yarnpkg.com/@fortawesome/free-solid-svg-icons/-/free-solid-svg-icons-5.13.0.tgz#44d9118668ad96b4fd5c9434a43efc5903525739"
+  integrity sha512-IHUgDJdomv6YtG4p3zl1B5wWf9ffinHIvebqQOmV3U+3SLw4fC+LUCCgwfETkbTtjy5/Qws2VoVf6z/ETQpFpg==
+  dependencies:
+    "@fortawesome/fontawesome-common-types" "^0.2.28"
+
+"@fortawesome/vue-fontawesome@^0.1.9":
+  version "0.1.9"
+  resolved "https://registry.yarnpkg.com/@fortawesome/vue-fontawesome/-/vue-fontawesome-0.1.9.tgz#d3af6d4e50f337327de90447fe35fa1e117a2fbe"
+  integrity sha512-h/emhmZz+DfB2zOGLWawNwXq82UYhn9waTfUjLLmeaIqtnIyNt6kYlpQT/vzJjLZRDRvY2IEJAh1di5qKpKVpA==
```
Comment (rvehall) [inline yarn.lock]: This still has the fontawesome stuff.

**Comment by chrissyalbert** [comment]
"Closes #12"

**Comment by rvehall** [comment]
Please add "Closes #12" to like this to this issue. Thanks!

File: yarn.lock (hunk 1 above)
Comment (chrissyalbert) [reply yarn.lock]: Just saw this. Thanks! Got rid of it!

File: src/views/Home.vue [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import Start from "@/components/Start.vue";
 
 export default {
+  //Why is this name "Resources"? Shouldn't it be "Home"? I'm confused
```
Comment (rvehall) [inline src/views/Home.vue]: Remove this comment (as it doesn't add anything) or update the name to Home and we're good.

**Review by rvehall — DISMISSED** [review]

File: src/views/Home.vue (hunk 2 above)
Comment (chrissyalbert) [reply src/views/Home.vue]: Okay, thank you! I changed it on the file on my computer, then pushed up the change, so it's asking you to review again. 
I'm learning a lot in this process, and I really appreciate your patience as I figure this all out!

**Review by rvehall — APPROVED** [review]

**Pull request merged**